db/migrate/20230318130154_create_discourse_rss_polling_rss_feeds.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateDiscourseRssPollingRssFeeds < ActiveRecord::Migration[7.0]
+  def change
+    create_table :discourse_rss_polling_rss_feeds do |t|
+      t.string :url, null: false
+      t.string :category_filter
+      t.string :author
+      t.integer :category_id
+      t.string :tags
```

Comment:
There's a discussion on dev about length limits for string columns. Would it make sense to apply some limits there too?